Tests/Integration/LoRaWan.Tests.Integration.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Required in order to be able to debug tests (running them works fine even without it).